src/components/AlphanumericInput/AlphanumericInput.svelte

```diff
@@ -60,11 +61,11 @@
   }
 </script>
 
-<div class={`numeric-input d-flex ${className}`}>
+<div class={`d-flex ${className}`}>
   {#each chunks as chunk, i}
     <input
       type="text"
-      class={`chunk form-control m-1 text-center fs-5 fw-bold ${chunkClass}`}
+      class={`chunk numeric-readable form-control m-1 text-center fs-5 fw-bold ${chunkClass}`}
```

Comment:
Why an extra class?

Just use `.alphanumeric-input .chunk` to style them.

```suggestion
      class={`chunk form-control m-1 text-center fs-5 fw-bold ${chunkClass}`}
```

Reply:
An extra class as it is used in two places for the same thing: a readable display of digits.

Reply:
I don't see the advantage `.alphanumeric-input .chunk` is way more self-explanatory than `.numeric-readable` because I know that `.alphanumeric-input` is probably coming from the `AlphanumericInput` component. But adding a new extra class make it hard to determine where it's coming from.

Reply:
>But adding a new extra class make it hard to determine where it's coming from.

We don't have an idientifier for custom global classes: `.global-` ?